recipe/meta.yaml

```diff
@@ -56,7 +56,10 @@ outputs:
       host:
         - cuda-version {{ cuda_version }}
       run:
-        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        # xref: conda-forge/libnvjitlink-feedstock#15
+        # we want to allow ex: libnvjitlink=12.6 installed in a cuda-version=12.3 environment,
```

Comment:
Do we want to allow an _older_ libnvjitlink than the rest of the CTK? What use case would that support? Seems like it could be problematic, but I’m not sure. We definitely want to relax this pinning but I’m not sure if it goes both directions.

Reply:
I think @bdice is right, we want something like: `cuda-version >=12,<=12.6` for CUDA 12.6 and `cuda-version >=12,<=12.4` for CUDA 12.4. This way, users never get a nvjitlink that is older than their cuda-version, but they *can* get a newer version. The pinning for this would be:

```
{{ pin_compatible("cuda-version", min_pin='x', max_pin="x.x") }}
```

Reply:
That's a good catch! @carterbox would you like to create an in-comment patch and apply it?